packages/synthetix-main/contracts/interfaces/IFundVault.sol

```diff
@@ -0,0 +1,78 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "./IFundVaultStorage.sol";
+
+/// @title Module for managing funds and assignments per account
+interface IFundVault is IFundVaultStorage {
```

Comment:
i think `IVault` is fine here np

Reply:
We might have vaults also for markets later. And also I'd like to have all Fund sections close and findable by name